packages/models/src/lib/notifications/unpublished-notification.model.ts

```diff
@@ -0,0 +1,22 @@
+import { NotificationKind } from './notification-kind';
+
+export interface UnpublishedNotification {
+    _id: string;
+
+    /**
+     * The ID of the thing (Work, Document, Blog, etc) that triggered this notification.
+     */
+    sourceId: string;
+
```

Comment:
The only real difference between an `UnpublishedNotification` and a `Notification` is the existence of the a `destinationUserId`--basically, a recipient.